lib/ansible/modules/cloud/linode/linode.py

```diff
@@ -597,18 +591,19 @@ def main():
                                                      'restarted']),
             api_key = dict(no_log=True),
             name = dict(type='str'),
-            alert_bwin_enabled = dict(type='bool', default=True),
-            alert_bwin_threshold = dict(type='int'),
-            alert_bwout_enabled = dict(type='bool', default=True),
-            alert_bwout_threshold = dict(type='int'),
-            alert_bwquota_enabled = dict(type='bool', default=True),
-            alert_bwquota_threshold = dict(type='int'),
-            alert_cpu_enabled = dict(type='bool', default=True),
-            alert_cpu_threshold = dict(type='int'),
-            alert_diskio_enabled = dict(type='bool', default=True),
-            alert_diskio_threshold = dict(type='int'),
-            backupweeklyday = dict(type='int'),
-            backupwindow = dict(type='int'),
+            alert_bwin_enabled = dict(type='bool', default=None),
```

Comment:
@s-hertel I changed the defaults back to how they were. In IRC it was pointed out that it was a subtle behavior change. So this whole branch has been to fix the issues and make it as backwards compatible as possible.

Now instead of passing in values if they're not needed, nothing is sent it which is how it was originally since those variables were never defined.